lib/constructor.js

```diff
@@ -0,0 +1,42 @@
+export default class BookList {
+  constructor() {
+    this.bookList = document.getElementById('book-list');
+    this.books = [];
+
+    const addBtn = document.getElementById('add-btn');
+    addBtn.addEventListener('click', () => this.addBook());
+  }
+
+  updateList(init) {
+    for (let i = init; i < this.books.length; i += 1) {
+      const bookDiv = document.createElement('div');
+      bookDiv.innerHTML += `
+        <span>"${this.books[i].title}" by 
+        ${this.books[i].author}</span>
+        <button onclick="myBooks.removeBook(${i})">Remove</button>
+        `;
+      bookDiv.classList.add('collection');
+      this.bookList.appendChild(bookDiv);
+    }
+  }
+
+  removeBook(bookNo) {
+    if (bookNo < 0) { return; }
+    this.books.splice(bookNo, 1);
+    localStorage.setItem('addBook', JSON.stringify(this.books));
+    this.bookList.innerHTML = '';
+    this.updateList(0);
+  }
+
+  addBook() {
```

Comment:
Good job working on the project requirements but I noticed whenever I try removing a book after adding the remove button is not functional and it due to the error in the screenshot below:
![Screenshot 2023-05-01 at 7 07 37 PM](https://user-images.githubusercontent.com/79506105/235494534-4c219fe2-ef5d-465e-bebb-ac199f172703.png)

It would be nice if you looked into this issue and fixed the error.